### PR TITLE
Use Debian Buster as base for slim image variant

### LINKED ISSUE #86: Is tehre a difference between slim-buster and slim-stretch?

### PR DESCRIPTION
**Pull request by hectcastro**
This change maintains alignment with the most recent stable Debian release, and
the base container image for the upstream Python container images.

Fixes https://github.com/azavea/docker-django/issues/86

---

**Testing**

- Confirm the upstream Python container images make use of Debian Buster

```console
$ docker run --rm -ti python:3-slim cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 10 (buster)"
NAME="Debian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```

- See: https://travis-ci.org/azavea/docker-django/builds/585385353

### CODE REVIEW

File: Dockerfile-slim.template [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM python:%%PYTHON_VERSION%%-slim-stretch
+FROM python:%%PYTHON_VERSION%%-slim-buster
```
Comment (rbreslow) [inline Dockerfile-slim.template:1]: I would also change the distribution of the PostgreSQL Apt repository.

**Review by rbreslow — CHANGES_REQUESTED** [review]

File: Dockerfile-slim.template (hunk 1 above)
Comment (hectcastro) [reply Dockerfile-slim.template:1]: Good catch. Resolved in https://github.com/azavea/docker-django/pull/87/commits/f1f71707ecccc645ec4ca179fb56b72c0ff8b84a and https://github.com/azavea/docker-django/pull/87/commits/40707c54d1111cf408954438686b169e69997774.

**Review by rbreslow — APPROVED** [review]
Cool. 👍

As a side note, I'm now subscribed to the [debian-announce](https://www.debian.org/MailingLists/subscribe) mailing list. It looks like they sent out an announcement about Buster sometime back in July.

**Comment by hectcastro** [comment]
> As a side note, I'm now subscribed to the debian-announce mailing list. It looks like they sent out an announcement about Buster sometime back in July.

Nice. I just did as well.

**Pull request merged**